uk.co.piggz.amazfish.json

```diff
@@ -0,0 +1,154 @@
+{
+  "app-id": "uk.co.piggz.amazfish",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-23.08",
+  "sdk": "org.kde.Sdk",
+  "command": "run-amazfish.sh",
+  "rename-icon": "harbour-amazfish-ui",
+  "rename-desktop-file": "harbour-amazfish-ui.desktop",
+  "rename-appdata-file": "harbour-amazfish-ui.appdata.xml",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--system-talk-name=org.bluez",
+    "--talk-name=org.mpris.MediaPlayer2.*",
+    "--talk-name=io.github.rinigus.PureMaps",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "nemo-qml-plugin-dbus",
+      "buildsystem": "qmake",
+      "builddir": true,
+      "cleanup": [
+        "/include"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/sailfishos/nemo-qml-plugin-dbus/archive/refs/tags/2.1.33.tar.gz",
+          "sha256": "08d1bdc5eac5eba2197f2c1e04a7e851437724331215059db1d68828dbf3b0a8"
+        },
+        {
+          "type": "patch",
+          "path": "nemodbus-qmake.patch"
+        }
+      ]
+    },
+    {
+      "name": "qtmpris",
+      "buildsystem": "qmake",
+      "builddir": true,
+      "cleanup": [
+        "/include"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/sailfishos/qtmpris/archive/refs/tags/1.0.6.tar.gz",
+          "sha256": "384eac76ec7d29d1de6fa93c45007ccd506d6cc0e4c69e19d4250e16cc0f34a2"
+        },
+        {
+          "type": "patch",
+          "path": "qtmpris-qmake.patch"
+        }
+      ]
+    },
+    {
+      "name": "libkdb3",
+      "buildsystem": "cmake",
```

Comment:
doesn't this work with `cmake-ninja` ?

Reply:
I don't know what is the difference. I will take a look into documentation and try it again.

Reply:
it's faster and recommended.